chart/templates/_helpers.yaml

```
@@ -607,6 +607,43 @@ Create the name of the cleanup service account to use
   {{- end }}
 {{- end }}
 
+{{define "scheduler_liveness_check_command"}}
+
+  {{- if semverCompare ">=2.0.0" .Values.airflowVersion }}
+  - sh
+  - -c
+  - |
+    CONNECTION_CHECK_MAX_COUNT=0 exec /entrypoint bash -c \
+    "export AIRFLOW__LOGGING__LOGGING_LEVEL=ERROR && \
+    airflow jobs check --job-type SchedulerJob --hostname $(hostname)"
```

```suggestion
    CONNECTION_CHECK_MAX_COUNT=0 AIRFLOW__LOGGING__LOGGING_LEVEL=ERROR \
    exec /entrypoint bash -c \
    "airflow jobs check --job-type SchedulerJob --hostname $(hostname)"
```
This is probably better...

Also, I don't think we need nested sh/bash, does this work?:
```suggestion
    CONNECTION_CHECK_MAX_COUNT=0 AIRFLOW__LOGGING__LOGGING_LEVEL=ERROR \
    exec /entrypoint airflow jobs check --job-type SchedulerJob --hostname $(hostname)
```

---

Yeah. Worked